readme.txt

```diff
@@ -0,0 +1,48 @@
+=== UCF Social Plugin ===
+Contributors: ucfwebcom
+Tags: ucf, social
+Requires at least: 4.7.3
+Tested up to: 4.7.3
+Stable tag: 1.0.0
+License: GPLv3 or later
+License URI: http://www.gnu.org/copyleft/gpl-3.0.html
+
+Provides a shortcode, functions, and default styles for displaying UCF social assets.
+
+
+== Description ==
+
+This plugin provides a shortcode, helper functions, and default styles for displaying social assets.  It is written to work out-of-the-box for non-programmers, but is also extensible and customizable for developers.
```

Comment:
We should add a note here about Athena-Framework/Bootstrap 3-4 and FontAwesome being required for this plugin to work properly out-of-the-box.